src/towncrier/test/test_check.py

```diff
@@ -476,14 +476,28 @@ def test_ignored_files(self, runner):
         """
         When `ignore` is set in config, files with those names are ignored.
```

Comment:
Maybe just update the scope of this test

```suggestion
        When `ignore` is set in config, files with those names are ignored.
        Configuration supports globbing expressions.
```